packages/google_atlas/lib/src/google_atlas.dart

```diff
@@ -164,20 +174,21 @@ class _GoogleMapsProviderState extends State<GoogleMapsProvider> {
     // https://github.com/flutter/flutter/issues/37185
     await Future<void>.delayed(Duration(seconds: 0));
     await _mapController.moveCamera(
-      _toGoogleCameraUpdate(cameraPosition, markers),
+      GoogleMaps.CameraUpdate.zoomTo(15),
```

Comment:
Is this hardcoding just an initial value and then the zoom level is changed by the activity on the map?

Reply:
yes